drv/transceivers-server/src/main.rs

```diff
@@ -194,20 +195,25 @@ impl ServerImpl {
     }
 
     fn update_leds(&mut self) {
-        // handle port LEDs
         let mut next_state = LogicalPortMask(0);
-        for (i, state) in self.led_states.0.into_iter().enumerate() {
-            let i = LogicalPort(i as u8);
-            match state {
-                LedState::On => next_state.set(i),
-                LedState::Blink => {
-                    if self.blink_on {
-                        next_state.set(i)
+
+        // We only turn transceiver LEDs on when Sidecar is in A0 since that is when there can be
+        // meaningful link activity happening. When outside of A0 we default the LEDs to off.
```

Comment:
nitpicky
```suggestion
        // We only turn transceiver LEDs on when Sidecar is in A0, since that is when there can be
        // meaningful link activity happening. When outside of A0, we default the LEDs to off.
```